client/my-sites/upgrades/components/domain-warnings/index.jsx

```diff
@@ -52,12 +62,20 @@ export default React.createClass( {
 	},
 
 	getPipe() {
-		const allRules = [ this.expiredDomains, this.expiringDomains, this.newDomains, this.unverifiedDomains, this.pendingGappsTosAcceptanceDomains ];
-		let rules;
+		let allRules = [
```

Comment:
`allRules` should be `const`, as in the old code. We could probably simplify the code a bit by getting rid of the `rules` variable and just returning when possible.